src/App.jsx

```diff
@@ -1,7 +1,25 @@
-export default function App() {
+import { useState } from "react";
+
+const App = () => {
+  const [count, setCount] = useState(0);
+  const incrementCount = () => setCount((prevCount) => prevCount + 1);
+
   return (
-    <div className="App">
-      <h1 className="text-2xl text-red-500">Hello React!</h1>
-    </div>
+    <main className="flex items-center justify-center h-screen">
+      <div className="flex flex-col items-center gap-4">
+        <p className="text-xl">
+          Button has been clicked:{" "}
```

Comment:
It is not ideal to use {" "} instead you can remove it, it's not doing anything.